README.md

```diff
@@ -6,6 +6,16 @@ This repository stores all the source files of [TiDB Docs at the PingCAP website
 
 If you find documentation issues, feel free to [create an Issue](https://github.com/pingcap/docs/issues/new/choose) to let us know or directly [create a Pull Request](/CONTRIBUTING.md#how-to-contribute) to help fix or update it.
 
+Currently, the official documentation supports two languages:
+
+- `en`: [documentation in English](/TOC.md)
```

Comment:
Let's give the official link on the PingCAP website, which is more friendly than the GitHub version.
```suggestion
- `en`: [documentation in English](https://docs.pingcap.com/tidb/stable)
```